composer.json

```diff
@@ -13,6 +13,9 @@
         "squizlabs/php_codesniffer": "3.0.0RC3"
     },
     "scripts": {
+        "post-install-cmd": [
+            "touch anime/Services/error.log",
+            "if [ ! -f anime/Services/state.json ]; then echo '{}' >anime/Services/state.json; fi"],
```

Comment:
Windows is a target platform? That's somewhat disappointing :(.
But understood.